addons/mail/static/src/models/notification_group/notification_group.js

```diff
@@ -101,6 +101,13 @@ function factory(dependencies) {
         id: attr({
             required: true,
         }),
+        /**
+         * Biggest `message.id` for this notification group. Used to sort by most
+         * recent message.
+         */
+        sequence: attr({
+            default: 0,
+        }),
```

Comment:
You should tell what this field is, not how it is computed with current code.
So here should only tell this is the sequence number of this group. This is used to order groups (lower sequence before higher sequence, same sequence order is arbitrary I suppose)